diff --git a/node_modules/@sveltejs/adapter-netlify/index.d.ts b/node_modules/@sveltejs/adapter-netlify/index.d.ts
index df05aec..5929816 100644
--- a/node_modules/@sveltejs/adapter-netlify/index.d.ts
+++ b/node_modules/@sveltejs/adapter-netlify/index.d.ts
@@ -1,5 +1,6 @@
 import { Adapter } from '@sveltejs/kit';
 
-declare function plugin(opts?: { split?: boolean; edge?: boolean }): Adapter;
-
-export = plugin;
+export default function plugin(opts?: {
+  split?: boolean;
+  edge?: boolean;
+}): Adapter;
