diff --git a/node_modules/@sveltejs/adapter-auto/index.d.ts b/node_modules/@sveltejs/adapter-auto/index.d.ts
index 62aee0f..90af12a 100644
--- a/node_modules/@sveltejs/adapter-auto/index.d.ts
+++ b/node_modules/@sveltejs/adapter-auto/index.d.ts
@@ -1,4 +1,3 @@
 import { Adapter } from '@sveltejs/kit';
 
-declare function plugin(): Adapter;
-export = plugin;
+export default function plugin(): Adapter;
