diff --git a/node_modules/@sveltejs/adapter-vercel/index.d.ts b/node_modules/@sveltejs/adapter-vercel/index.d.ts
index 1566ac1..f2ce972 100644
--- a/node_modules/@sveltejs/adapter-vercel/index.d.ts
+++ b/node_modules/@sveltejs/adapter-vercel/index.d.ts
@@ -1,10 +1,9 @@
 import { Adapter } from '@sveltejs/kit';
 
 type Options = {
-	edge?: boolean;
-	external?: string[];
-	split?: boolean;
+  edge?: boolean;
+  external?: string[];
+  split?: boolean;
 };
 
-declare function plugin(options?: Options): Adapter;
-export = plugin;
+export default function plugin(options?: Options): Adapter;
